### ISSUE TITLE
cwd target doesn't work with ibazel

### ISSUE THREAD

**Issue by alexeagle** [theoremlp/rules_multitool#67]
A customer pointed out that the pattern documented on the README
https://github.com/theoremlp/rules_multitool?tab=readme-ov-file#running-tools-in-the-current-working-directory
doesn't work correctly with some programs like `ibazel` (bazel-watcher) because zombie processes get left behind.
We think this post from @lberki is related https://groups.google.com/g/bazel-discuss/c/I6TWUv8AI5o/m/4nsrPIiogVcJ?pli=1
I think this can be reproduced by running some server target under `./tools/ibazel //my/server:target`, then when you ctrl-c (SIGTERM) you'll still see the server is running.

So we changed our `aspect init` https://github.com/aspect-build/aspect-workflows-template/pull/87
to still stamp out a script that DOES function correctly:

```
#!/bin/sh
target="@multitool//tools/$(basename "$0")"
bazel 2>/dev/null build "$target" && exec $(bazel info execution_root)/$(bazel 2>/dev/null cquery --output=files "$target") "$@"
```

Personally I think `bazel run` is still just busted in a way rules_multitool shouldn't try to fix.

**Comment by mark-thm** [comment]
Ah, that's a bummer. I wasn't able to find a current issue on Bazel's GitHub issues tracker, I'll file one.

**Comment by mark-thm** [comment]
I tried to build a minimal reproduction of `bazel run` propagating SIGINT but was able to successfully trap SIGINT using the example from the linked mailing list. It seems like `bazel run` (at least in Bazel 7.0.0+) is propagating SIGINT.

I do see that ibazel is leaving dangling processes when run through multitool's cwd target, but it's less clear to me that bazel run is broken?

**Comment by alexeagle** [comment]
Interesting, @walkerburgin confirmed they saw a problem but I don't know what Bazel version. Do you have time to make a repro?

**Comment by mark-thm** [comment]
Running latest ibazel directly (no bazel, shell, or multitool involved) also leaves dangling processes on a ctrl+c.

**Comment by alexeagle** [comment]
Yeah it's an ibazel bug, it so happens that calling it through `bazel run` exposes it while invoking the `ibazel` binary directly does not. So I think the only change here should be a warning in that README not to use that pattern with ibazel.
